core/blockly.ts

```diff
@@ -429,7 +429,7 @@ Names.prototype.populateProcedures = function (
 };
 // clang-format on
 
-export * from './interfaces/i_navigable.js';
+export * from './flyout_navigator.js';
```

Comment:
Is anything actually being exported from here?

Reply:
Yes, the `FlyoutNavigator` class, which developers may wish to subclass.